routes/loginRoute.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var router = express.Router();
+var path = require('path');
+
+/* GET home page. */
+router.get('/', function(req, res) {
+    const cookies = req.cookies;
+    if(cookies.auth_token){
+        res.sendFile(path.join(__dirname,'../public/displayPage.html'));
+    }
+    else{
+        res.sendFile(path.join(__dirname,'../public/login.html'));
```

Comment:
send a redirect to corresponding page, but not file..

you should see /login in route and also login page when there is not cookie.